src/signals/IncidentMap/components/AddressLocation/AddressLocation.tsx

```diff
@@ -32,8 +33,11 @@ export const AddressLocation = ({
   return (
     <Wrapper>
       <Heading as="h4">Zoom naar adres</Heading>
+      <Label htmlFor="searchAddressBar">searchAddressBar</Label>
```

Comment:
aa is see, its default is hidden!

Reply:
Right, but what should I change? It's not the most elegant solution, but by not setting the inner HTML I got an error message.

Reply:
@SireeKoolenWijkstra please see comment below with docs.